### DISCUSSION TITLE
Git Flow을 위한 폴더 네이밍을 제안합니다.

### DISCUSSION THREAD

**Discussion by IGhost-P** [EasyAndBeauty/DailyReceipt-Flutter#17] [Ideas]
# 배경
- git flow 전략은 모두가 아시겠지만~ git flow를 좀더 잘 활용할수 있는 네이밍을 제안합니다. (https://medium.com/@abhay.pixolo/naming-conventions-for-git-branches-a-cheatsheet-8549feca2534)

# 제안 내용
- 만약 feature 관련 issue라면 (예시로 폴더를 만드는 피처파면)
-feature/make-folder 로 네이밍을 지으면 feature그룹안에 자연스럽게 make-folder가 들어갑니다!

**Comment by jayjeong8** [comment]
좋습니다! 공유해주신 글 내용이 좋네요 ㅎㅎ 
빠른 참고를 위해 요약 내용을 기록합니다! 

#### 기본 규칙
- 하이픈(-)으로 단어 구분 `feature/new-login`
- 소문자, 숫자, 하이픈만 사용 (a-z, 0–9, -)
- 연속 하이픈은 사용하지 않음 (css 네이밍처럼 `feature--new-login`로 작성하지 않음)
- 이름을 하이픈으로 끝내지 않음 `feature-new-login-`
- 수행할 작업을 나타내는 설명적이고 간결한 이름 사용

#### 브랜치 접두사
- `feature/`
- `bugfix/`
- `hotfix/`
- `release/`
- `docs/`


---

프로젝트 규모가 작기도 하고 초기 단계라서 브랜치 접두사 및 git flow는 `feature/`, `bugfix/`  정도 선에서 가져가면 좋을 것 같아요!
